.env

```diff
@@ -6,3 +6,5 @@ APP_DEFAULT_LANGUAGE=en
 
 # a comma separated list of languages that are enabled (this only works for languages that are enabled in the app)
 APP_ENABLED_LANGUAGES=en,es
+
+APP_GOOGLE_SITE_VERIFICATION_ID=joKVm0-rBn0Xqsqrdu_MPgQU2_jJvKSnqwEwodBvnhk
```

Comment:
Let's not include it in the .env file (because that will get picked up by any forked repos) and instead still feed it in via vars and env in the github action. It's not a sensitive value, but nice to implement in a way that other developers can fork and set their own values without create git noise.